dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/task/ILogicTask.java

```diff
@@ -31,4 +34,7 @@ public interface ILogicTask {
 
     TaskExecutionContext getTaskExecutionContext();
 
+    default List<Property> getVarPool() {
+        return getTaskParameters().getVarPool();
+    }
```

Comment:
I don't agree we need to add this method in `ILogicTask`, the Task SPI doesn't need to add this method, otherwise we need to add a lot of get method.